modules/apps/hooks/hooks.go

```diff
@@ -0,0 +1,20 @@
+package ibc_hooks
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	channeltypes "github.com/cosmos/ibc-go/v5/modules/core/04-channel/types"
+	ibcexported "github.com/cosmos/ibc-go/v5/modules/core/exported"
+)
+
+type TransferHooks interface {
```

Comment:
I don't know if we need to call these transfer hooks. Technically this implementation should work with any IBC application to add hooks to it correct? So maybe it should be `IBCApplicationHooks`